core/web/nodes_controller.go

```diff
@@ -76,9 +76,9 @@ func (n *nodesController[R]) Index(c *gin.Context, size, page, offset int) {
 		// fetch nodes for chain ID
 		// backward compatibility
 		var rid relay.ID
-		err := rid.UnmarshalString(id)
+		err = rid.UnmarshalString(id)
```

Comment:
This one was a bug. The linter flagged on 84, because due to this mistake here, that assignment was ineffectual. By reusing the error var from the enclosing scope, we ensure that errors from 84 are passed to the response in 93, just like on 74.